src/elvis_style.erl

```diff
@@ -516,6 +541,21 @@ node_line_limits(FunctionNode) ->
 %% Private
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+%% Variables name
+check_variables_name(_Regex, []) -> [];
+check_variables_name(Regex, [Variable | RemainingVars]) ->
+    VariableNameStr = atom_to_list(ktn_code:attr(name, Variable)),
+    case re:run(VariableNameStr, Regex) of
+        nomatch when VariableNameStr == "_" ->
+            check_variables_name(Regex, RemainingVars);
+        nomatch ->
+            Msg = ?VARIABLE_NAMING_CONVENTION_MSG,
+            Info = [VariableNameStr, Regex],
+            Result = elvis_result:new(item, Msg, Info, 1),
```

Comment:
Please provide the location of the var in the error report so that it is easier to find. You can check the following code [here](https://github.com/inaka/elvis/blob/master/src/elvis_style.erl#L509) to know how to extract the `location` information from a node.